models/mpas_atm/readme.rst

```diff
@@ -468,16 +455,16 @@ Mac and NCAR IBM supercomputer (yellowstone).
    necessary.
 
 Conversions
-===========
+-----------
 
 A Welcome Development
----------------------
+~~~~~~~~~~~~~~~~~~~~~
 
 MPAS files no longer beed to be converted to DART formatted files, they can be
 read in directly from a input file list!
 
 Analysis File NetCDF header
----------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Should we add a clarifying remark that the variables marked 'available in DART'  are metadata variables that are always available in DART and do not need to explicitly made part of the DART state?
Otherwise, it is not clear what these comments mean in regard to setting the DART state via ``mpas_state_variables``

Reply:
done